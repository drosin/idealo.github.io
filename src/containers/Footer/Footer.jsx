import React from 'react';
import './Footer.scss';

import Socials from '../../components/Socials/Socials'

export default (props) => {
  return (
    <footer className='footer__container'>
      <div className='footer__content'>
        <Socials />
        <p className='footer__links'>
          <a href='http://www.axelspringer.de/artikel/Impressum_40386.html'>Impressum</a>
          <a href='http://www.axelspringer.de/artikel/Hinweise-zum-Datenschutz_40410.html'>Hinweise zum Datenschutz</a>
        </p>
      </div>
    </footer>
  );
}