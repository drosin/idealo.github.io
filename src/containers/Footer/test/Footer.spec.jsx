import React from 'react';
import renderer from 'react-test-renderer';
import Footer from '../Footer';

describe('Footer component', () => {
    it('should render as expected', () => {
        const renderedComponent = renderer.create(
            <Footer />
        );
        expect(renderedComponent.toJSON()).toMatchSnapshot();
    })
});