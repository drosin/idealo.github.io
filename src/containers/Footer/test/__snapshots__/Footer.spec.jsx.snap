// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component should render as expected 1`] = `
<footer
  className="footer__container"
>
  <div
    className="footer__content"
  >
    <p
      className="socials__container undefined"
    >
      <a
        className="socials__item"
        href="https://www.xing.com/companies/axelspringerse"
      >
        <span
          aria-hidden={true}
          className="fa fa-xing"
        />
      </a>
      <a
        className="socials__item"
        href="https://de.linkedin.com/company/axel-springer-se"
      >
        <span
          aria-hidden={true}
          className="fa fa-linkedin"
        />
      </a>
      <a
        className="socials__item"
        href="https://www.instagram.com/axel_springer_karriere/"
      >
        <span
          aria-hidden={true}
          className="fa fa-instagram"
        />
      </a>
      <a
        className="socials__item"
        href="https://www.facebook.com/AxelSpringerKarriere/"
      >
        <span
          aria-hidden={true}
          className="fa fa-facebook"
        />
      </a>
    </p>
    <p
      className="footer__links"
    >
      <a
        href="http://www.axelspringer.de/artikel/Impressum_40386.html"
      >
        Impressum
      </a>
      <a
        href="http://www.axelspringer.de/artikel/Hinweise-zum-Datenschutz_40410.html"
      >
        Hinweise zum Datenschutz
      </a>
    </p>
  </div>
</footer>
`;
